test_data/pil/expand_fixed.pil

```diff
@@ -0,0 +1,172 @@
+namespace std::prelude;
+    let true: bool = "" == "";
+    let false: bool = !true;
+
+namespace std::check;
+    /// This is a built-in function taking a string argument and terminating
+    /// evaluation unsuccessfully with this argument as explanation.
+    /// This symbol is not an empty array, the actual semantics are overridden.
+    let panic: string -> ! = [];
+
+    /// Checks the condition and panics if it is false.
+    /// IMPORTANT: Since this does not generate any constraints, the verifier will not
+    /// check these assertions. This function should only be used to verify
+    /// prover-internal consistency.
+    /// The panic message is obtained by calling the function `reason`.
+    /// Returns an empty array on success, which allows it to be used at statement level.
+    let assert: bool, (-> string) -> () = |condition, reason| if !condition { panic(reason()) } else { () };
+
+namespace std::prover;
+    /// Returns the current number of rows, sometimes known as the "degree".
+    let degree: -> int = [];
+
+namespace std::utils;
+    /// Evaluates to folder(...folder(folder(initial, f(0)), f(1)) ..., f(length - 1)),
+    /// i.e. calls f(0), f(1), ..., f(length - 1) and combines the results
+    /// using the function `folder`, starting with the value `initial`.
+    ///
+    /// See `sum` for an example use.
+    let<T1, T2>
+        fold: int, (int -> T1), T2, (T2, T1 -> T2) -> T2 = |length, f, initial, folder|
```

Comment:
I would propose to turn this test into an `.asm` test so that you don't have to copy the std